packages/ts-utils/src/json.ts

```diff
@@ -0,0 +1,4 @@
+/** Normalized JSON with 4 spaces a as indent */
+export const toJson = (obj: unknown): string => {
```

Comment:
I'd rather call this `asIndentedJsonString` (or `asPrettyJsonString`) because `obj` already _is_ JSON, or you couldn't call `JSON.stringify` with it anyway.

In addition, I'd like to implement a function `asMinimalJsonString` that just does `JSON.stringify`.

Then, we should replace all `JSON.stringify(<obj>)` with `asMinimalJsonString`, and all other `JSON.stringify`s with `asPrettyJsonString`.